fs/metrics.h

```diff
@@ -2,13 +2,134 @@
 
 #include "rocksdb/env.h"
 #include "utilities/trace/bytedance_metrics_reporter.h"
+#include "zbd_stat.h"
+#include <map>
 
 namespace ROCKSDB_NAMESPACE {
 
+
+struct BDZenFSMetrics : public ZenFSMetrics {
```

Comment:
We should move this whole `metrics.h` outside of the zenfs (maybe `env/env_zenfs_metrics.h`?)